### PR TITLE
added PRUDENCE regions to defined scientific regions

### PR DESCRIPTION
**Pull request by ruthlorenz**
- [ ] Passes `isort . && black . && flake8`
 - [ ] Fully documented, including `whats-new.rst`

### CODE REVIEW

File: docs/defined_scientific.rst [hunk 1]
```diff
@@ -114,9 +115,23 @@ Ocean
...
+PRUDENCE Regions
+================
+
+The PRUDENCE regions were defined in the PRUDENCE project as European sub-areas for regional climate model output and are often used in European climate studies. They contain 8 regions, Alps (AL), British Isles (BI), Eastern Europe (EA), France (FR), Iberian Peninsula (IP), Mediterranean (MD), Mid-Europe (ME), and Scandinavia (SC). 
+
+.. warning:: The region FR and ME overlapp, hence 2D masks will not be correct. Use 3D masks, createmask for each region separately and then concatenate, see https://github.com/regionmask/regionmask/issues/228
+
+.. ipython:: python
+
+    regionmask.defined_regions.prudence.plot();
```
Comment (mathause) [inline docs/defined_scientific.rst:127]:
```suggestion
    regionmask.defined_regions.prudence.plot(resolution="50m");
```

File: docs/defined_scientific.rst [hunk 2]
```diff
@@ -114,9 +115,23 @@ Ocean
     plt.tight_layout()
 
 
+PRUDENCE Regions
+================
+
+The PRUDENCE regions were defined in the PRUDENCE project as European sub-areas for regional climate model output and are often used in European climate studies. They contain 8 regions, Alps (AL), British Isles (BI), Eastern Europe (EA), France (FR), Iberian Peninsula (IP), Mediterranean (MD), Mid-Europe (ME), and Scandinavia (SC). 
+
+.. warning:: The region FR and ME overlapp, hence 2D masks will not be correct. Use 3D masks, createmask for each region separately and then concatenate, see https://github.com/regionmask/regionmask/issues/228
```
Comment (mathause) [inline docs/defined_scientific.rst:123]:
```suggestion
.. warning:: The region FR and ME overlap, hence 2D masks will not be correct. Use ``mask_3D`` to create masks for each region separately and then concatenate, see `#228 <https://github.com/regionmask/regionmask/issues/228>`__.
```

File: docs/whats_new.rst [hunk 3]
```diff
@@ -25,6 +25,8 @@ Deprecations
 New regions
 ~~~~~~~~~~~
 
+- Added :py:attr:`prudence` regions for Europe from `Christensen and Christensen, 2007, <https://link.springer.com/article/10.1007/s10584-006-9210-7>`_.
```
Comment (mathause) [inline docs/whats_new.rst:28]:
```suggestion
- Added :py:attr:`prudence` regions for Europe from `Christensen and Christensen, 2007, <https://link.springer.com/article/10.1007/s10584-006-9210-7>`_ (:pull:`276`). By `Ruth Lorenz <https://github.com/ruthlorenz>`_.

```

**Review by mathause — COMMENTED** [review]
Looks good - thanks! I added some minor comments. And could you add the regions to the dict in 

https://github.com/regionmask/regionmask/blob/0872147942af9070a6d33b4e417ecc8cf7320159/regionmask/tests/utils.py#L43

the int is the number of sub-regions.

**Comment by mathause** [comment]
Ahh I messed this up, sorry. Could you force push (`git push --force origin master`) your master again and open another PR. I think I should be able to take it from there. (Or I can open a new PR from my account - I think you will still get credit but I am not entirely sure.)

**Comment by ruthlorenz** [comment]
Done, I opened a new pull request

**Pull request closed without merging**